[2024-05-26T02:01:46.472+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-26T02:01:46.501+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: DAG_catedra_airflow.second-step-el scheduled__2024-05-25T02:35:00+00:00 [queued]>
[2024-05-26T02:01:46.512+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: DAG_catedra_airflow.second-step-el scheduled__2024-05-25T02:35:00+00:00 [queued]>
[2024-05-26T02:01:46.513+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-05-26T02:01:46.537+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): second-step-el> on 2024-05-25 02:35:00+00:00
[2024-05-26T02:01:46.545+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=259) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-26T02:01:46.548+0000] {standard_task_runner.py:63} INFO - Started process 261 to run task
[2024-05-26T02:01:46.549+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'DAG_catedra_***', 'second-step-el', 'scheduled__2024-05-25T02:35:00+00:00', '--job-id', '60', '--raw', '--subdir', 'DAGS_FOLDER/dag_clase.py', '--cfg-path', '/tmp/tmpwxigva8l']
[2024-05-26T02:01:46.551+0000] {standard_task_runner.py:91} INFO - Job 60: Subtask second-step-el
[2024-05-26T02:01:46.603+0000] {task_command.py:426} INFO - Running <TaskInstance: DAG_catedra_airflow.second-step-el scheduled__2024-05-25T02:35:00+00:00 [running]> on host da1b21f42040
[2024-05-26T02:01:46.713+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='DAG_catedra_***' AIRFLOW_CTX_TASK_ID='second-step-el' AIRFLOW_CTX_EXECUTION_DATE='2024-05-25T02:35:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-05-25T02:35:00+00:00'
[2024-05-26T02:01:46.715+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-26T02:01:46.722+0000] {logging_mixin.py:188} INFO - The connection to the DB was succesful
[2024-05-26T02:01:46.737+0000] {logging_mixin.py:188} INFO - Query completed
[2024-05-26T02:01:46.743+0000] {logging_mixin.py:188} INFO - execute_query_result_set()
[2024-05-26T02:01:46.750+0000] {logging_mixin.py:188} INFO - The connection to the DB was succesful
[2024-05-26T02:01:46.755+0000] {logging_mixin.py:188} INFO - Inserts were comitted
[2024-05-26T02:01:46.756+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-26T02:01:46.757+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-26T02:01:46.766+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=DAG_catedra_***, task_id=second-step-el, run_id=scheduled__2024-05-25T02:35:00+00:00, execution_date=20240525T023500, start_date=20240526T020146, end_date=20240526T020146
[2024-05-26T02:01:46.803+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-26T02:01:46.822+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-26T02:01:46.827+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
